addons/binding/org.openhab.binding.smappee/src/main/java/org/openhab/binding/smappee/internal/SmappeeHandlerFactory.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.binding.smappee.internal;
+
+import static org.openhab.binding.smappee.SmappeeBindingConstants.THING_TYPE_SMAPPEE;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.openhab.binding.smappee.handler.SmappeeHandler;
+
+/**
+ * The {@link SmappeeHandlerFactory} is responsible for creating things and thing
+ * handlers.
+ *
+ * @author Niko Tanghe - Initial contribution
+ */
+public class SmappeeHandlerFactory extends BaseThingHandlerFactory {
+
+    private final static Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections.singleton(THING_TYPE_SMAPPEE);

```

Comment:
This constant is also declared in `SmappeeBindingConstants`

Reply:
removed from SmappeeBindingConstants since it's only used in SmappeeHandlerFactory